addons/website_sale/views/snippets/snippets.xml

```diff
@@ -87,7 +87,7 @@
                 </we-select>
                 <we-select string="⌙ Position">
                     <we-button data-set-ribbon-position="left">Left</we-button>
-                    <we-button data-set-ribbon-position="right">Right</we-button>
+                    <we-button data-set-ribbon-position="right">Right (only on grid view)</we-button>
```

Comment:
You have to update the POT file for the translated term update in a stable version.